packages/team-management/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@opensrp/team-management",
+  "version": "0.0.0",
+  "description": "Team management components for opensrp web",
+  "main": "dist/index.js",
+  "types": "dist/types",
+  "files": [
+    "dist"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "https://github.com/OpenSRP/web/",
+  "scripts": {
```

Comment:
its missing build script

@KipSigei  @kelvin-muchiri @p-netm  @haseemVD @kahummer  we need to start writing cross platform script

i am creating an issues for it

Reply:
@KipSigei Let's have the package.json match the one in the [template package](https://github.com/OpenSRP/web/blob/master/packages/template/package.json)

Reply:
@kelvin-muchiri addressed